### ISSUE TITLE
activity

### ISSUE THREAD

**Issue by Velosify** [Orbiit/ugwa2#1]
hmmm i should do something

**Comment by SheepTester** [comment]
Yes you should; admittedly I'm not sure what you can do, but perhaps we can figure something else

WITH SYNERGY

**Comment by Velosify** [comment]
This video was sponsored by [Synergy](https://symless.com/synergy)! Synergy lets you share a single mouse and keyboard between two or more computers!

**Comment by SheepTester** [comment]
Alarming

- Sean

**Comment by Velosify** [comment]
very

**Comment by SheepTester** [comment]
[ok here are some jobs](https://github.com/Orbiit/ugwa2/issues/assigned/Velosify)

**Comment by Velosify** [comment]
im still very inactive wheeeeeee
